v3/setup/mesos-credentials.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -x
+
+source /etc/environment
+if [ "${NODE_ROLE}" != "control" ] && [ "${NODE_ROLE}" != "worker"]; then
+    exit 0
+fi
+
+# ensure that we have sane defaults for credentials
+etcdctl get /mesos/config/username
+if [[ $? = 4 ]]; then
+    etcdctl set /mesos/config/username ethos
+fi
+etcdctl get /mesos/config/password
+if [[ $? = 4 ]]; then
+    etcdctl set /mesos/config/password password
```

Comment:
Not sure how I feel about this - on one hand the stack will come up if a user forgets to pass in a password. On the other hand, we're hardcoding insecure defaults.